demos/theme/index.html

```diff
@@ -457,11 +457,113 @@ <h2 class="mdc-typography--headline demo-component-section__heading">
           </div>
         </figure>
       </section>
+
+      <section class="demo-component-section">
+        <i id="radio" class="demo-anchor-with-toolbar-offset"></i>
+        <h2 class="mdc-typography--headline demo-component-section__heading">
+          Radio Button
+          <a href="#radio" class="demo-component-section__permalink" title="Permalink to the theme demo for the radio button component">#</a>
+        </h2>
+
+        <figure class="demo-radio-group">
+          <div class="mdc-form-field demo-radio-form-field">
+            <div class="mdc-radio">
+              <input class="mdc-radio__native-control" type="radio" id="demo-radio-1" checked name="demo-radio-group-1">
+              <div class="mdc-radio__background">
+                <div class="mdc-radio__outer-circle"></div>
+                <div class="mdc-radio__inner-circle"></div>
+              </div>
+            </div>
+            <label id="demo-radio-1-label" for="demo-radio-1">Default 1</label>
+          </div>
+          <div class="mdc-form-field demo-radio-form-field">
+            <div class="mdc-radio">
+              <input class="mdc-radio__native-control" type="radio" id="demo-radio-2" name="demo-radio-group-1">
+              <div class="mdc-radio__background">
+                <div class="mdc-radio__outer-circle"></div>
+                <div class="mdc-radio__inner-circle"></div>
+              </div>
+            </div>
+            <label id="demo-radio-2-label" for="demo-radio-2">Default 2</label>
+          </div>
+        </figure>
+
+        <figure class="demo-radio-group">
+          <div class="mdc-form-field demo-radio-form-field">
+            <div class="mdc-radio demo-radio--secondary">
+              <input class="mdc-radio__native-control" type="radio" id="demo-radio-3" checked name="demo-radio-group-2">
+              <div class="mdc-radio__background">
+                <div class="mdc-radio__outer-circle"></div>
+                <div class="mdc-radio__inner-circle"></div>
+              </div>
+            </div>
+            <label id="demo-radio-3-label" for="demo-radio-3">Custom 1</label>
```

Comment:
Change Custom 1 and Custom 2 to Secondary 1 and Secondary 2, then Custom 3 and Custom 4 to Custom 1 and Custom 2? Since all of them are pairs, the 4 aren't in one group.

Reply:
Done.